static_website/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "account" {
+}
+
+variable "region" {
+}
+
+variable "environment" {
+}
+
+variable "nubis_domain" {
+  default = "nubis.allizom.org"
+}
+
+variable "technical_owner" {
+  default = "infra-aws@mozilla.com"
+}
+
```

Comment:
I'm not quite sure what you mean here, the technical_owner variable is only used as a tag in this module and thus isn't really technically needed. I'm just adding it to follow some tagging that we have on s3 buckets for example. If this is not needed I'm happy to remove it